feat: Document new app service export error metrics (#1237)
edgexfoundry/edgex-docs: docs_src/microservices/application/AdvancedTopics.md

- `MessagesReceived` - This is a **counter** metric that counts the number of messages received by the application service. Includes invalid messages.

- `InvalidMessagesReceived ` -  This is a **counter** metric that counts the number of invalid messages received by the application service. 

- `HttpExportSize  ` -  This is a **histogram** metric that collects the size of data exported via the built-in [HTTP Export pipeline function](../BuiltIn/#http-export). The metric data is not currently tagged due to breaking changes required to tag the data with the destination endpoint. This will be addressed in a future EdgeX 3.0 release.

- `HttpExportErrors` - **(New)** This is a **counter** metric that counts the number errors encounter when exporting via HTTP. 

- `MqttExportSize  ` -  This is a **histogram** metric that collects the size of data exported via the built-in [MQTT Export pipeline function](../BuiltIn/#mqtt-export). The metric data is tagged with the specific broker address and topic.

> - `MqttExportErrors` -  **(New)** This is a **counter** metric that counts the number errors encounter when exporting via MQTT. 

Reviewer: ```suggestion
- `MqttExportErrors` -  **(New)** This is a **counter** metric that counts the number of errors encountered when exporting via MQTT. 
```

- `HttpExportSize  ` -  This is a **histogram** metric that collects the size of data exported via the built-in [HTTP Export pipeline function](../BuiltIn/#http-export). The metric data is not currently tagged due to breaking changes required to tag the data with the destination endpoint. This will be addressed in a future EdgeX 3.0 release.

> - `HttpExportErrors` - **(New)** This is a **counter** metric that counts the number errors encountered when exporting via HTTP. 

Reviewer: ```suggestion
- `HttpExportErrors` - **(New)** This is a **counter** metric that counts the number of errors encountered when exporting via HTTP. 
```

- `MqttExportSize  ` -  This is a **histogram** metric that collects the size of data exported via the built-in [MQTT Export pipeline function](../BuiltIn/#mqtt-export). The metric data is tagged with the specific broker address and topic.

> - `MqttExportErrors` -  **(New)** This is a **counter** metric that counts the number errors encountered when exporting via MQTT. 

Reviewer: ```suggestion
- `MqttExportErrors` -  **(New)** This is a **counter** metric that counts the number of errors encountered when exporting via MQTT. 
```
